pkgs/applications/video/screenkey/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, distutils_extra
+, setuptools-git
+, intltool
+, pygtk
+, libX11
+, libXtst
+}:
+buildPythonApplication rec {
+  name = "screenkey-0.9";
+
+  src = fetchFromGitHub {
+    owner = "wavexx";
+    repo = "screenkey";
+    rev = name;
+    sha256 = "14g7fiv9n7m03djwz1pp5034pffi87ssvss9bc1q8vq0ksn23vrw";
+  };
+
+  buildInputs = [
+    distutils_extra
+    setuptools-git
+    intltool
```

Comment:
Why didn't you include all dependencies from https://www.thregr.org/~wavexx/software/screenkey/#installation-and-basic-usage?
Also, some of these should go into `nativeBuildInputs` since they are only needed for building. See https://nixos.org/nixpkgs/manual/#ssec-stdenv-attributes.

Reply:
`slop` is an optional runtime dependency and is not required for normal operation. (Even "install dependencies" command for Ubuntu/Debian does not list it.)

As for `nativeBuildInputs`, I agree. (Developed the expression on top of the stable channel and it does not feature `nativeBuildInputs` as a separate argument in `mk-python-derivation` yet.)

Reply:
:+1: